pkg/apis/core/v1alpha1/types_overridepolicy.go

```diff
@@ -81,12 +81,46 @@ type TargetClusters struct {
 	ClusterAffinity []ClusterSelectorTerm `json:"clusterAffinity,omitempty"`
 }
 
+// Overriders contains a list of override patch
+// The order in which the override patches take effect is:
+// - Image
+// - JsonPatch
 type Overriders struct {
+	Image []ImageOverrider `json:"image,omitempty"`
```

Comment:
Should add comments and // +optional here

Reply:
The code has been updated.